pages/desktop/user.py

```diff
@@ -119,8 +120,19 @@ def click_update_account(self):
         self.selenium.find_element(*self._update_account_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: self.update_message == "Profile Updated")
 
-    def change_hide_email_state(self):
-        self.selenium.find_element(*self._hide_email_checkbox).click()
+    def hide_email_address(self):
+        email_checkbox = self.selenium.find_element(*self._email_checkbox)
+        WebDriverWait(self.selenium, self.timeout).until(expected_conditions.visibility_of(email_checkbox));
+        if not email_checkbox.is_selected():
+                self.selenium.find_element(*self._email_checkbox).click()
```

Comment:
The action of the method is to hide the email via the checkbox. If I do what you suggest then it will fail the test if the email is already hidden, which doesn't seem right to me - because it isn't failing to hide the email, it's failing to be unhidden before the test started. If it fails before the test starts then the failure should be somewhere else.